install-nix.sh

```diff
@@ -34,7 +34,7 @@ installer_options=(
 )
 
 # only use the nix-daemon settings if on darwin (which get ignored) or systemd is supported
-if [[ $OSTYPE =~ darwin || -e /run/systemd/system ]]; then
+if [[ $OSTYPE =~ darwin || -e /run/systemd/system && $INPUT_SINGLE_USER == "" ]]; then
```

Comment:
Maybe we should check if this is `!= true`? Otherwise the input option has semantics like: any value makes it true.